index.js

```diff
@@ -63,7 +63,41 @@ function encodeConstructorArgs(inputs) {
   return bytecode;
 }
 
+/**
+ * Decodes function args.
+ *
+ * @param {Object} contractABI - ABI of contract whose args to decode
+ * @param {string} bytecode - full call args bytecode
+ * @returns {Object} decodedArgs - Object representing decoded args with name, type, and data fields
+ */
+function decodeFuncionArgs(contractABI, bytecode) {
```

Comment:
Could you fix the typos? (Missing a "t")

Reply:
my bad, fixed.